packages/dev/core/src/XR/webXRExperienceHelper.ts

```diff
@@ -11,6 +11,17 @@ import { Logger } from "../Misc/logger";
 import { UniversalCamera } from "../Cameras/universalCamera";
 import { Quaternion, Vector3 } from "../Maths/math.vector";
 
+
+/**
+ * Options for setting up XR spectator camera.
+ */
+export interface XRSpectatorModeOption {

```

Comment:
Just a thought, if you already added an options object (which is great) - maybe add "proffered eye" which will decide whether to take from the first rig camera or the second (if available).

Also, we try to separate our interfaces from the WebXR ones. the native WebXR interfaces all start with XR, and babylon's all start with WebXR.

Reply:
Resolved by a645095ed9f74295a06a0c1a81a38f414f11f216